### ISSUE TITLE
Floats cannot be used as query params

### ISSUE THREAD

**Issue by lukaszdudek-codibly** [aio-libs/yarl#491]
Hello!

Consider the following:

```
Python 3.7.7 (default, Jun  2 2020, 15:59:15) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import yarl
>>> yarl.__version__
'1.5.1'
>>> yarl.URL("localhost").with_query(a=0.0)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ljbd/.pyenv/versions/teems/lib/python3.7/site-packages/yarl/_url.py", line 970, in with_query
    new_query = self._get_str_query(*args, **kwargs)
  File "/home/ljbd/.pyenv/versions/teems/lib/python3.7/site-packages/yarl/_url.py", line 931, in _get_str_query
    query = "&".join(self._query_seq_pairs(quoter, query.items()))
  File "/home/ljbd/.pyenv/versions/teems/lib/python3.7/site-packages/yarl/_url.py", line 901, in _query_seq_pairs
    yield quoter(key) + "=" + quoter(cls._query_var(val))
  File "/home/ljbd/.pyenv/versions/teems/lib/python3.7/site-packages/yarl/_url.py", line 912, in _query_var
    "of type {}".format(v, type(v))
TypeError: Invalid variable type: value should be str or int, got 0.0 of type <class 'float'>
```

I'd expect that floats are correctly encoded as query parameters.

**Comment by lukaszdudek-codibly** [comment]
Oh, I've just found https://github.com/aio-libs/yarl/issues/65 - if that's your feeling still @asvetlov , I'll close this issue.

**Comment by asvetlov** [comment]
Well, we support `int` currently, practicality beats purity.

What do you expect for `url.update_query(a=0.0)`? `?a=0` or `?a=0.0`?
Floats are comparable with integers in Python (but Pascal has a different rule for this IIRC).
Values in URLs are strings, `?a=0` is not equal to `?a=0.0`.
Integers have no such ambiguity.

P.S.
Regarding booleans -- there is no standard representation for bools but different sites use `yes`, `true`, `True`, `1` and others.
That's why `bool` is not supported.

**Comment by lukaszdudek-codibly** [comment]
Sure. I'd expect that  `url.update_query(a=0.0)` renders what was passed, so `?a=0.0`. Whoever's using it that way has to have full awareness of the inequality between `?a=0` & `?a=0.0`.

**Comment by asvetlov** [comment]
Ok. Please feel free to make a pull request.

**Comment by asvetlov** [comment]
Fixed by #505
